### ISSUE TITLE
No highlighting of current page in the sidebar navigation

### ISSUE THREAD

**Issue by kitation** [UKHomeOffice/hods-poc#27]
There should be a visual indicator as to which page a user is on in the sidebar navigation. Also all the links have aria-current set to true which makes the screen reader say every page is the current page.

**Comment by daniel-ac-martin** [comment]
**(Notes to self):**
The `aria-current` issue seems to be due to the use of react-router's NavLink component: https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/NavLink.md

It appears we could fix this by either avoiding NavLink, in favour of the simpler Link component, or by adding some logic in an [`isActive`](https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/NavLink.md#isactive-func) prop.

**Comment by daniel-ac-martin** [comment]
The second part of this issue

> Also all the links have aria-current set to true which makes the screen reader say every page is the current page.

should now be fixed. See: https://hods-poc.netlify.app/components?name=Footer

**Comment by kitation** [comment]
Yep aria-current issue is fixed.

**Comment by daniel-ac-martin** [comment]
@kitation: Would you mind checking this one again when you get a chance?

**Comment by kitation** [comment]
Yep looks good!
